src/components/playlist-countdown/_playlist-countdown.scss

```diff
@@ -8,11 +18,15 @@
     right: 0;
     margin: 16px;
     transform: translateY(0px);
-    transition: transform ease-in 100ms, right 500ms 500ms, opacity 400ms, bottom 300ms;
+    transition: transform ease-in 100ms, opacity 400ms, bottom 300ms;
     cursor: pointer;
 
+    &.slideIn{
+      animation: slideIn 1000ms
+    }
+
     &.hidden {
-      right: -160px;
+      transition: opacity 50ms;
```

Comment:
This causes a bad animation once moving from show to hidden 

![Kaltura-Player-V3-_1_](https://user-images.githubusercontent.com/17685520/70864444-95dc9380-1f5a-11ea-90c7-8cecc6b5adf1.gif)